bundles/org.openhab.binding.webexteams/src/main/java/org/openhab/binding/webexteams/internal/api/AccessTokenResponse.java

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.webexteams.internal.api;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * @author Tom Deckers - Initial contribution
+ */
+@NonNullByDefault
+class AccessTokenResponse {
+    @SerializedName("access_token")
+    @Nullable
+    private String accessToken;
+    @SerializedName("expires_in")
+    @Nullable
+    private Integer expiresIn;
+    @SerializedName("refresh_token")
+    @Nullable
+    private String refreshToken;
+    @SerializedName("refresh_token_expires_in")
+    @Nullable
+    private Integer refreshTokenExpiresIn;
```

Comment:
The code becomes more readable when adding these null annotations to the types:

```suggestion
    private @Nullable Integer refreshTokenExpiresIn;
```

Reply:
Done.